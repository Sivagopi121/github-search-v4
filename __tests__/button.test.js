import React from 'react';
import renderer from 'react-test-renderer';
import Button from '../src/Button/index'

test('App renders correctly', () => {
  const tree = renderer
    .create(
      <Button />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
