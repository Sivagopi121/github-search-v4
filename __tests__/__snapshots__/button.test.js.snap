// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<button
  className="undefined Button Button_black"
  type="button"
/>
`;
